examples/c/minimal_legacy.bpf.c

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2020 Facebook */
+#define BPF_NO_GLOBAL_DATA
+#include "vmlinux.h"
```

Comment:
given we are not using CO-RE (and legacy systems won't have vmlinux BTF), let's include UAPI headers we need instead? Line linux/bpf.h